openslides_backend/action/mixins/import_mixins.py

```diff
@@ -66,8 +66,11 @@ def __init__(
         self.name_to_ids: Dict[SearchFieldType, List[Dict[str, Any]]] = defaultdict(
             list
         )
-        for name, _ in name_entries:
-            self.name_to_ids[name] = []
+        for name, name_entry in name_entries:
+            if name in self.name_to_ids:
+                self.name_to_ids[name].append(name_entry)
+            else:
+                self.name_to_ids[name] = []
```

Comment:
does this not cause problems in comparison lines 107 - 116? It should at the very least make the else clause in line 111 obsolete. Might as well delete it, if that's the intention.

Reply:
No, that works quite good together. 
If line 71 will be reached, it is a duplicate and line 110 will append the second entry to recognize the duplicate.